### PR TITLE
[Example] Support atrous convolution in PoseNet example

### PR DESCRIPTION
**Pull request by Wenzhao-Xiang**
**No ATROUS_DEPTHWISE_CONV2D:**
image
**ATROUS_DEPTHWISE_CONV2D:**
image

Performance: 
Only 20~30 ms speed up in WASM, and almost no speed up in WebGL.

### CODE REVIEW

**Comment by Wenzhao-Xiang** [comment]
For NNAPI don't support atrous (depthwise)conv2d now, keep the original NO-ATROUS model.
You can choose the model with dilated options as below:

image

And the **default** dilated value is **false**.

**Comment by huningxin** [comment]
Nice work! I will review it ASAP. Thanks much for the immediate update!

File: examples/posenet/gui.js [hunk 1]
```diff
@@ -2,6 +2,7 @@ let guiState;
 guiState = {
   algorithm: 'multi-pose',
   model: 1.01,
+  dilated: false,
```
Comment (huningxin) [inline examples/posenet/gui.js]: `use_atrous_conv`?

File: examples/posenet/src/PoseNet.js [hunk 2]
```diff
@@ -1,12 +1,13 @@
...
-  constructor(modelArch, version, outputStride, inputShape, type, cacheMap, backend, prefer) {
+  constructor(modelArch, version, dilated, outputStride, inputShape, type, cacheMap, backend, prefer) {
     this._modelArch = modelArch;
     this._model = null;
     this._compilation;
     this._execution;
     this._tensorIds = [];
     this._operandIndex = 0;
     this._version = version;
+    this._dilated = dilated;
```
Comment (huningxin) [inline examples/posenet/src/PoseNet.js]: For clarity, I would suggest to change variable name to `_use_atrous_conv`. Does it sound?

Comment (Wenzhao-Xiang) [reply examples/posenet/src/PoseNet.js]: Looks easier to understand! Thanks!

**Comment by huningxin** [comment]
Thanks @Wenzhao-Xiang ! I am going to merge it.

BTW, @pinzhenx , do you think we can apply this method to DeepLab dilated model?

**Pull request merged**